### PR TITLE
Fp21 87 footer

### PR DESCRIPTION
**Pull request by hassou2n**
# Description

I added the social media links, I modified the copyright text, and I changed the footer to the new design.

# How to test?

Run the client and server

# Checklist

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [x] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [x] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: packages/client/src/components/Footer/Footer.style.css [hunk 1]
```diff
@@ -50,46 +41,37 @@ footer {
 
 .footer-container {
   width: 1220px;
-  height: 140px;
+  height: 155px;
   background-color: rgba(245, 245, 245, 0.05);
```
Comment (somayehdvr) [inline packages/client/src/components/Footer/Footer.style.css]: The background color is not compatible with the Figma design.

**Review by somayehdvr — APPROVED** [review]
I suggest changing the background color for the footer container to match the design although it was not a part of your work in this branch.

File: packages/client/src/components/Footer/Footer.style.css (hunk 1 above)
Comment (hassou2n) [reply packages/client/src/components/Footer/Footer.style.css]: I changed the background color, Thx

**Review by shpomp — CHANGES_REQUESTED** [review]
This is what I see:
image
And from, f.x., this, it seems like a deliberate choice:
image

Not sure what the idea is here 🤷🏽‍♀️

**Comment by hassou2n** [comment]
@shpomp I fixed the background color, I really don't know what happened, because I fixed it before

**Comment by shpomp** [comment]
> @shpomp I fixed the background color, I really don't know what happened, because I fixed it before

I see it now. Cannot get it to run with the latest commit, but that is on my side. Adding the changes from the last commit, it looks good 👍🏽

**Review by shpomp — APPROVED** [review]

**Pull request merged**